### DISCUSSION TITLE
正しくアイテムの編集が反映できることを確認するテスト

### DISCUSSION THREAD

**Discussion by chikowaka** [ushmz/subsuke#34] [Product backlog]
誰が　　　：
何をしたい：

テスト要件
前提条件　：
手順　　　：
入力値　　：
期待する値：

**Comment by ushmz** [comment]
ロジックは `subscription_item_bloc.dart`
https://github.com/ushmz/subsuke/blob/c734f9f5c9e672c855288edea321b673838940bf/lib/blocs/subscription_item_bloc.dart#L96-L99
